Resources/Prototypes/SS220/Recipes/Construction/Furniture/chairplastic.yml

```
@@ -0,0 +1,16 @@
+- type: construction
+  name: plastic chair
+  id: ChairPlastic
+  graph: SeatPlastic
+  startNode: start
+  targetNode: chairPlastic
+  category: construction-category-furniture
+  description: Вы сидите на этом. Либо по вашей воле, либо по принуждению, либо ради мотивации и силы!
```

```suggestion
  description: You're sitting on it. Either by your will, or by compulsion, or for the sake of motivation and power!
```
Пока не добавят локали для рецептов, нужно либо делать всё на инглише, либо переводить вообще всё (а без локалей это как то кринжово, хз, да и работа впустую, ибо локали рано или поздно сделают, а работу придётся откатывать), чтобы не выбивалось